.github/workflows/CI.yml

```
@@ -19,7 +19,7 @@ jobs:
         rust:
           - "stable"
         cpp:
-          - llvm
+          - llvm-15.0.2
```

setup-cpp now defaults to LLVM 15 by default

```suggestion
          - llvm
```